src/app/[locale]/components/organisms/events/CreateEvent.tsx

```diff
@@ -177,8 +175,6 @@ const CreateEvent = () => {
       style: toastStyle,
       iconTheme: toastIconTheme,
     })
-
-    cleanDataStorage()
```

Comment:
removing those function it will not clean whole values, this feature requieres delete evriting except the list of selected countries